eogrow/utils/testing.py

```diff
@@ -89,26 +90,34 @@ def calculate_statistics(folder: str, config: StatCalcConfig) -> JsonDict:
         elif content_path.endswith(".npy"):
             stats[content] = _calculate_numpy_stats(np.load(content_path, allow_pickle=True), config)
         elif content_path.endswith((".geojson", ".gpkg")):
-            stats[content] = _calculate_vector_stats(gpd.read_file(content_path), config)
+            stats[content] = _calculate_vector_stats(gpd.read_file(content_path, engine="pyogrio"), config)
         elif content_path.endswith(".parquet"):
-            try:
-                data = gpd.read_parquet(content_path)
-            except Exception:
-                data = _load_as_geoparquet(content_path)
-            stats[content] = _calculate_vector_stats(data, config)
+            # don't sample random rows due to non-determinism in parquet row order
+            stats[content] = _get_parquet_stats(content_path, replace(config, num_random_values=0))
```

Comment:
i was thinking of leaving this to the user

Reply:
failed to notice that the config is passed in `compare_content`, removed

Reply:
recent improvement :)